tools/cbmc/proofs/HTTP/global_state_HTTP.c

```diff
@@ -23,6 +29,9 @@ void *safeMalloc(size_t xWantedSize) {
  * responseHandle, requestHandle and connectionHandle similarly.
  */
 
+// TODO: Replace this with a model allowing unconstrained sizes,
+// or at least allowing a small set of differing sizes.
```

Comment:
That is a good idea for the data part of the structure.

Reply:
That is what I meant.  I propose we do this after the release as part of the next package.

Reply:
I've written a better comment.